source/tutorials/pd-mice.rst

```diff
@@ -405,27 +408,31 @@ We’ll start by using the ``qiime diversity core-metrics-phylogenetic`` method
    -  Unweighted UniFrac distance
    -  Weighted UniFrac distance
 
-There is a very good discussion of diversity metrics and their meanings in a `qiime forum by Stephanie Orchanian`_.
+There is a very good discussion of diversity metrics and their meanings in a `forum post by Stephanie Orchanian`_.
 
-This method wraps several other methods, and it’s worthwhile to note that the steps in ``qiime diversity core-metrics-phylogenetic`` can be executed independently.
+The ``qiime diversity core-metrics-phylogenetic`` method wraps several other methods, and it's worthwhile to note that the steps can also be executed independently.
 
-One important consideration for diversity calculations is the rarefaction depth. Above, we used the alpha rarefaction visualization and the sample summary visualization to pick a rarefaction depth. So, for these analyses, we’ll use a depth of 1000 sequences per sample.
+One important consideration for diversity calculations is the rarefaction depth. Above, we used the alpha rarefaction visualization and the sample summary visualization to pick a rarefaction depth. So, for these analyses, we'll use a depth of 2020 sequences per sample.
 
 .. command-block::
 
    qiime diversity core-metrics-phylogenetic \
      --i-table ./dada2_table.qza \
      --i-phylogeny ./tree.qza \
      --m-metadata-file ./metadata.tsv \
-     --p-sampling-depth 2000 \
+     --p-sampling-depth 2020 \
```

Comment:
I would like to return to the 2000 depth we had before. You can't do nice multiple rarefaction if you're at the same depth as the shallowest sample (a big issue I have with moving pictures which I thought I'd nicely managed to dodge here) and non round numbers as 18 kinds of obnoxious

Reply:
> You can't do nice multiple rarefaction if you're at the same depth as the shallowest sample

Can you clarify this? I am not sure I follow.